dids/src/main/kotlin/web5/sdk/dids/didcore/Service.kt

```diff
@@ -0,0 +1,80 @@
+package web5.sdk.dids.didcore
+
+/**
+ * Service is used in DID documents to express ways of communicating with
+ * the DID subject or associated entities.
+ * A service can be any type of service the DID subject wants to advertise.
```

Comment:
again, another spec link? https://www.w3.org/TR/did-core/#services

Reply:
oh I see you're linking to the spec in the `@property` comment below (and that may have been true to previous comments) ... do whatevs you think is best, I'm moving quickly